docs/api_guide/quickstart.rst

```diff
@@ -80,19 +80,55 @@ when extracting streams more than once. You start by creating a
 .. code-block:: python
 
     >>> from streamlink import Streamlink
-    >>> session = Streamlink()
+    >>> session = Streamlink({"optional-session-option": 123})
 
-You can then set options like this:
+On the session instance, you can set additional options like so:
 
 .. code-block:: python
 
     >>> session.set_option("stream-timeout", 30)
+    >>> session.options.set("stream-timeout", 30)
 
-and extract streams like this:
+See :py:class:`StreamlinkOptions <streamlink.session.options.StreamlinkOptions>` for all available session options.
+
+
+Fetching streams
+^^^^^^^^^^^^^^^^
+
+Streams can be fetched in two different ways.
+
+The first example will automatically try to find a matching plugin and available streams from the input URL:
 
 .. code-block:: python
 
     >>> streams = session.streams("URL")
 
+See :py:meth:`Streamlink.streams() <streamlink.session.Streamlink.streams>` for more.
+
+It however doen't allow you to set any plugin options which might be necessary in order to access streams, e.g. when
```

Comment:
I don't think "It" should be used here to avoid confusion. The exact name should be used. "`session.streams` doesn't allow you to..."